pkg/machines/components/create-vm-dialog/createVmDialog.jsx

```diff
@@ -749,6 +749,11 @@ export class CreateVmAction extends React.Component {
         this.close = this.close.bind(this);
     }
 
+    componentWillMount() {
+        cockpit.script('virt-install --version', null, { err: 'message' })
+                .then(() => this.setState({ virtInstallAvailable: true }));
```

Comment:
Can you please initialize this to `null` in the ctor, so that the full structure of state can be seen there?